src/Sql/Sql/help/Refresh-AzSqlServerExternalGovernanceStatus.md

```diff
@@ -0,0 +1,131 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
```

Comment:
https://learn.microsoft.com/powershell/module/az.sql/refresh-azsqlserverexternalgovernancestatus

Reply:
The link is giving 404. Is that the issue?